public/app/features/plugins/loader/cache.ts

```diff
@@ -1,17 +1,26 @@
 import { clearPluginSettingsCache } from '../pluginSettings';
 
-const cache: Record<string, string> = {};
+const cache: Record<string, CacheablePluginEntry> = {};
 const initializedAt: number = Date.now();
 
 type CacheablePlugin = {
   path: string;
   version: string;
+  isAngular?: boolean;
 };
 
-export function registerPluginInCache({ path, version }: CacheablePlugin): void {
+type CacheablePluginEntry = {
+  version: string;
+  isAngular?: boolean;
+};
```

Comment:
What's the reason to both have `CacheablePlugin` and `CacheablePluginEntry`? Could they be "merged"?

Reply:
I've merged them. 👍